tensorflow_datasets/image_classification/caltech.py

```diff
@@ -37,17 +37,19 @@
 contains bounding boxes.
 """
 _LABELS_FNAME = "image_classification/caltech101_labels.txt"
-_URL = "http://www.vision.caltech.edu/Image_Datasets/Caltech101/"
+_URL = "https://drive.google.com/uc?export=download&id=137RyRjvTBkBiIfeYBNZBtViDHQ6_Ewsp"
+_HOMEPAGE = "http://www.vision.caltech.edu/Image_Datasets/Caltech101/"
 _IMAGES_FNAME = "101_ObjectCategories.tar.gz"
```

Comment:
I think we won't we needing this anymore

Reply:
Sure, I've removed it in the next commit.